package.json

```diff
@@ -43,5 +43,8 @@
     "type": "git",
     "url": "git://github.com/mozilla/pdf.js.git"
   },
-  "license": "Apache-2.0"
```

Comment:
I don't understand what's happening here.
I didn't change the package.json file and still it shows as edited and the build on travis-ci seems to have failed.